.github/workflows/main.yml

```diff
@@ -8,7 +8,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [ubuntu-latest]
+        os: [ubuntu-latest, macos-latest, windows-2019]
```

Comment:
I found that using `windows-2019` instead of `windows-2022`, which happened to become `windows-latest` just recently, significantly improves runtime, so I also submitted https://github.com/crate/crate/pull/12344.